File: 0075-sort-colors/0075-sort-colors.java
class Solution {
    public void sortColors(int[] nums) {
        int a=0;
        int i=0;
        int size=(nums.length)-1;
        while(i<=size){
            if(nums[i]==2){
                int temp = nums[i];
                nums[i]=nums[size];
                nums[size]=temp;
                size--;
            }
            else if(nums[i]==0){
                int temp = nums[i];
                nums[i]=nums[a];
                nums[a]=temp;
                a++;
                i++;
            }
            else{
                i++;
            }
        }
        // return nums;
    }
}

File: 0075-sort-colors/0075-sort-colors.py
class Solution:
    def sortColors(self, nums: List[int]) -> None:
        a=0
        b=len(nums)-1
        i=0
        while i<=b:
            if nums[i]==0:
                nums[i],nums[a]=nums[a],nums[i]
                a+=1
                i+=1
            elif nums[i]==2:
                nums[i],nums[b]=nums[b],nums[i]
                b-=1
            else:
                i+=1
                